Apps/WebClient/src/NewClientApp/src/components/modal/AppTourComponent.vue

```diff
@@ -0,0 +1,222 @@
+            "@/assets/images/tour/web/at-notifications-web.png",
+            import.meta.url
+        ).href,
+        imageAlt: "Notifications centre web demo",
+    },
+];
+
+const { mdAndUp } = useDisplay();
+
+const slideIndex = ref(0);
+const isVisible = ref(false);
+
+const slides = computed<TourSlide[]>(() => {
+    // (MOBILE) Requires new assets for mobile, currently returning only the desktopSlides
+    //    1. Reimplement checks for store's isMobile getter
+    //    2. Introduce mobileSlides array
+    //    3. Uncomment return statement below, naturally remove the current return desktopSlides.
+    // return this.isMobile ? mobileSlides : desktopSlides;
+    return desktopSlides;
+});
+
+const currentSlide = computed<TourSlide | undefined>(() => {
+    return slides.value.length > 0 ? slides.value[slideIndex.value] : undefined;
+});
+
+const isFinalSlide = computed<boolean>(() => {
+    return slideIndex.value === slides.value.length - 1;
+});
+
+const isFirstSlide = computed<boolean>(() => {
+    return slideIndex.value === 0;
+});
+
+function hideModal(): void {
+    slideIndex.value = 0;
+    isVisible.value = false;
+}
+
+function next(): void {
+    slideIndex.value++;
+}
+function previous(): void {
+    slideIndex.value--;
+}
+</script>
+
+<template>
+    <v-dialog
+        id="app-tour"
+        v-model="isVisible"
+        body-class="p-0"
+        content-class="overflow-hidden"
+        size="lg"
+        persistent
+        no-click-animation
+    >
+        <template v-slot:activator="{ props }">
+            <HgIconButtonComponent v-bind="props" data-testid="app-tour-button">
+                <v-badge
+                    color="red"
+                    :model-value="highlightTourChangeIndicator"
+                >
+                    <v-icon icon="fas fa-lightbulb" />
+                </v-badge>
+            </HgIconButtonComponent>
+        </template>
+        <v-row justify="center" no-gutters>
+            <v-card max-width="675px">
+                <v-card-text class="pa-0 mh-50">
+                    <v-carousel
+                        id="tourCarousel"
+                        v-model="slideIndex"
+                        :continuous="false"
+                        :show-arrows="false"
+                        hide-delimiter-background
+                        :height="mdAndUp ? 480 : 250"
+                    >
+                        <v-carousel-item
+                            v-for="slide in slides"
+                            :key="slide.title"
+                            :src="slide.imageUri"
+                            :alt="slide.imageAlt ?? slide.title"
+                            transition="scroll-x-reverse-transition"
+                        />
+                    </v-carousel>
+                </v-card-text>
+                <template #actions>
+                    <v-container>
+                        <v-row v-if="currentSlide">
+                            <v-col>
```

Comment:
Row & col wrapper here don't seem to be needed - could be replaced with `<div>`.